modal/image.py

```diff
@@ -120,6 +127,36 @@ def _make_pip_install_args(
     return args
 
 
+def _get_image_builder_version(client_version: ImageBuilderVersion) -> ImageBuilderVersion:
+    if config_version := config.get("image_builder_version"):
+        version = config_version
+        if (env_var := "MODAL_IMAGE_BUILDER_VERSION") in os.environ:
```

Comment:
Not super important, but it might be nice to have `Config` itself have a method to pretty-print the source of a given value